app/src/main/java/edu/uc/hickmadc/customnotifications/adapters/GroupsAdapter.kt

```diff
@@ -12,7 +12,10 @@ import edu.uc.hickmadc.customnotifications.dto.Group
 class GroupsAdapter(private val dataSet: ArrayList<Group>) :
     ListAdapter<Group, RecyclerView.ViewHolder>(GroupDiffCallback()) {
 
-    // Creates list rows / views
+    /**
+     * Creates list rows / views
+     * @return populates the ViewHolder
```

Comment:
Added return annotation.

Reply:
This is the proper formatting for comments over a method.